apps/web/screens/Category/Category.tsx

```diff
@@ -139,17 +124,14 @@ const Category: Screen<CategoryProps> = ({
     setHash(hashString)
   }, [Router])
 
-  const sidebarCategoryLinks = categories.map((c) => ({
-    title: c.title,
-    active: c.slug === Router.query.slug,
-    href: makePath('ArticleCategory', '/[slug]'),
-    as: makePath('ArticleCategory', c.slug),
-  }))
-
-  const categoryOptions = categories.map((c) => ({
-    label: c.title,
-    value: c.slug,
-  }))
+  const sidebarCategoryLinks = categories.map(({ __typename, title, slug }) => {
+    return {
+      title,
+      typename: __typename,
+      active: slug === Router.query.slug,
+      slug: [slug],
+    }
+  })
```

Comment:
I'm not sure how the category pages work, but in Article pages we need the parent article for the first link, is that maybe not the case here? or is this maybe a different setup?

Reply:
Yes, but it's not the case here. In the navbar here we display all categories, no subcategories or something of that sort.